meetings/minutes/weekly-2020-07-17.md

```diff
@@ -0,0 +1,20 @@
+# Weekly meeting
+
+**2020-07-17**
+
+Chair: @imtbkcat
+
+## New features
+
+* Async commit
+  - TODO
+  
+* Commit transactions with concurrent DDLs [18098](https://github.com/pingcap/tidb/pull/18098) @cfzjywxk
+  - [18447](https://github.com/pingcap/tidb/pull/18447) support commit transaction with add/drop column and add/drop index DDLs, in review.
+
+* Optimizing memory usage of large transaction in TiDB @bobotu
```

Comment:
```suggestion
* [Introducing an optimized transaction buffer](https://github.com/pingcap/tidb/pull/18650) @bobotu
  -  Because the patchset is too large to review, I have split it into multiple small patchsets. You can track current progress at [here](https://github.com/pingcap/tidb/projects/54#card-42001319)
```